src/lib_gui/qt/project_wizard/content/QtProjectWizardContentPreferences.cpp

```diff
@@ -552,6 +567,13 @@ void QtProjectWizardContentPreferences::save()
 
 	appSettings->setLoggingEnabled(m_loggingEnabled->isChecked());
 	appSettings->setVerboseIndexerLoggingEnabled(m_verboseIndexerLoggingEnabled->isChecked());
+	if (m_logPath &&
+			m_logPath->getText().toStdWString() != appSettings->getLogDirectoryPath().wstr()) {
```

Comment:
I think you can remove the line break after "&&" and please move the `{` into the next line

Reply:
done